src/_P039_Thermosensors.ino

```diff
@@ -405,9 +423,21 @@ boolean Plugin_039(byte function, struct EventStruct *event, String& string)
           }
           {
             addFormNote(F("Set Thermocouple type for MAX31856"));
-            const String Toptions[8]      = { F("B"), F("E"), F("J"), F("K"), F("N"), F("R"), F("S"), F("T") };
-            const int    ToptionValues[8] = { 0, 1, 2, 3, 4, 5, 6, 7 };
-            addFormSelector(F("Thermocouple type"), F("P039_tctype"), 8, Toptions, ToptionValues, P039_TC_TYPE);
+            const String Toptions[10]      = { F("B"), F("E"), F("J"), F("K"), F("N"), F("R"), F("S"), F("T"), F("VM16"), F("VM32") };
+            const int    ToptionValues[10] = { 0, 1, 2, 3, 4, 5, 6, 7, 8, 12};
```

Comment:
Just a check... this sequence looks odd. Is it correct?

Reply:
Yes - as i use the value directly as input to the register, this is should be correct and reflecting the required value. Maybe a final review and a comment could help here...i wll add.

Reply:
Solved and indeed some minor corrections needed ;-)